### DISCUSSION TITLE
This user account is blocked - Pro teams

### DISCUSSION THREAD

**Discussion by commanderx25** [vercel/community#3187] [Help] [answered]
### Question

![Screenshot 2023-07-11 085049](https://github.com/vercel/community/assets/42400985/9e1cb5a6-19f4-401c-8a9c-49564f2b74f5)
 How to solve this issue, my github connected pro team account is locked, I can't login

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) for help with login issues. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
